atuin-common/src/utils.rs

```diff
@@ -51,6 +52,22 @@ pub fn data_dir() -> PathBuf {
     data_dir.join("atuin")
 }
 
+pub fn get_days_from_month(year: i32, month: u32) -> i64 {
+    NaiveDate::from_ymd(
+        match month {
+            12 => year + 1,
+            _ => year,
+        },
+        match month {
+            12 => 1,
+            _ => month + 1,
+        },
+        1,
+    )
+    .signed_duration_since(NaiveDate::from_ymd(year, month, 1))
+    .num_days()
+}
+
```

Comment:
Does chrono/time not expose any better solution for this? 😭

Reply:
Chrono does not 🙃 and I'd rather avoid adding another dependency for one single function that's pretty easy to implement - though if we end up needing more from that crate it can be added

Reply:
haha makes sense! Rust library landscape is constantly changing 😂